<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рабочая область</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../src_img/icon.png</normaloff>../../src_img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(191, 185, 182);</string>
  </property>
  <widget class="QTableView" name="database_tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QSplitter" name="tableManipulate_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>581</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>52</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>27</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="add_button">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ДОБАВИТЬ ЗАПИСЬ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>УДАЛИТЬ ЗАПИСЬ</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="tableView_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>23</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="usersView_button">
    <property name="text">
     <string>«Покупатели»</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purchaseView_button">
    <property name="text">
     <string>«Покупки»</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productView_button">
    <property name="text">
     <string>«Продукты»</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supplyView_button">
    <property name="text">
     <string>«Поставки»</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>src_img/return.png</normaloff>src_img/return.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
